src/core/common/logging.hpp

```diff
@@ -1215,6 +1232,39 @@ const char *otLogRegionToString(otLogRegion aRegion);
     otPlatLog(                                                              \
         aLogLevel,                                                          \
         aRegion,                                                            \
+        "%s%s: " aFormat OPENTHREAD_CONFIG_LOG_SUFFIX,                      \
```

Comment:
Suggestion: How about the following format: `"[%s]%s"`  so we get the log level in `[...]`...